### ISSUE TITLE
IconMenu not rendering MenuItems

### ISSUE THREAD

**Issue by flake** [mrphu3074/react-material-ui#10]
> var MenuItem = MUI.MenuItem // is it right way?

because some issues from npm show that new menuitem should be used from /material-ui/lib/menus/menu-item [#1346](https://github.com/callemall/material-ui/issues/1346)

> <AppBar
>           title={this.state.appTitle}
>           onLeftIconButtonTouchTap={ SideNavActions.toggle }
>           iconElementRight={
>             <IconMenu iconButtonElement= {<IconButton iconClassName="material-icons" iconStyle={ {color: "#fff"} }>more_vert</IconButton>} >
>               <MenuItem primaryText="Refresh" index={0} />
>              <MenuItem primaryText="Send feedback" index={1} />
>               <MenuItem primaryText="Settings" index={2} />
>               <MenuItem primaryText="Help" index={3} />
>               <MenuItem primaryText="Sign out" index={4} />
>             </IconMenu>
>           } />

IconMenu is empty. not showing MenuItems.

**Comment by mrphu3074** [comment]
```
  //We're working on migrating some of our components to use a new implementation of menus.
    //If you'd like to use the new menu before our migration is complete, please directly
    //require them like this:

    let MenuItem = require('material-ui/lib/menus/menu-item');
    let MenuDivider = require('material-ui/lib/menus/menu-divider');
```

They working on Menu's components. So i fixed this issue in ver 0.1.5
Please use

```
MUI.Libs.Menu
MUI.Libs.MenuItem
MUI.Libs.MenuDivider
```

From version 0.1.5, my package supported SSR :dancer:  
I created a demo 

http://material-ui-ssr.meteor.com

**Comment by flake** [comment]
@mrphu3074 Thank you :)
